src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        return List.of(
+                new Truck(),
+                new Truck(),
+                new Truck()

```

Comment:
use parameterized constructors. at the moment, you've added fields to this class, but not initialized them